qiita_db/test/test_processing_job.py

```diff
@@ -741,13 +741,13 @@ def test_hide(self):
     def test_shape(self):
         jids = {
             # Split libraries FASTQ
-            '6d368e16-2242-4cf8-87b4-a5dc40bb890b': (27, 31),
+            '6d368e16-2242-4cf8-87b4-a5dc40bb890b': (27, 31, 116),
             # Pick closed-reference OTUs
-            '80bf25f3-5f1d-4e10-9369-315e4244f6d5': (27, 31),
+            '80bf25f3-5f1d-4e10-9369-315e4244f6d5': (27, 31, 0),
```

Comment:
Why is the input size zero? Is it because the FASTA files associated with this artifact are empty?

Reply:
Yup, they are empty ...